includes/object-cache.php

```diff
@@ -1062,13 +965,9 @@ public function fetch_info() {
         if ( defined( 'WP_REDIS_CLUSTER' ) ) {
             $connectionID = current( $this->build_cluster_connection_array() );
 
-            if ( $this->determine_client() === 'credis' ) {
-                $info = $this->redis->info();
-            } else {
-                $info = ($this->determine_client() === 'predis')
-                    ? $this->redis->getClientFor( $connectionID )->info()
-                    : $this->redis->info( $connectionID );
-            }
+            $info = ($this->determine_client() === 'predis')
+                ? $this->redis->getClientFor( $connectionID )->info()
+                : $this->redis->info( $connectionID );
```

Comment:
Let's only focus on pings on PhpRedis and Predis. No need to test Credis or HHVM since we are deprecating it.

Reply:
So this code should stay like this in its current situation without checking `creids`